platform/android/java_godot_wrapper.cpp

```diff
@@ -213,6 +214,23 @@ bool GodotJavaWrapper::is_dark_mode() {
 	}
 }
 
+Color GodotJavaWrapper::get_accent_color() {
+	if (_get_accent_color) {
+		JNIEnv *env = get_jni_env();
+		ERR_FAIL_NULL_V(env, Color(0, 0, 0, 0));
+		int accentColor = env->CallIntMethod(godot_instance, _get_accent_color);
+
+		// Convert ARGB to RGBA
+		int alpha = (accentColor >> 24) & 0xFF;
+		int red = (accentColor >> 16) & 0xFF;
+		int green = (accentColor >> 8) & 0xFF;
+		int blue = accentColor & 0xFF;
```

Comment:
```suggestion
		int accent_color = env->CallIntMethod(godot_instance, _get_accent_color);

		// Convert ARGB to RGBA.
		int alpha = (accent_color >> 24) & 0xFF;
		int red = (accent_color >> 16) & 0xFF;
		int green = (accent_color >> 8) & 0xFF;
		int blue = accent_color & 0xFF;
```
Please use `snake_case` for c++